### PR TITLE
OPS: remove npx (closes #3078)

### LINKED ISSUE #3078: Remove npx calls&dependency as used packages by it already installed
The npx is called in:
 
1. The first npx usage `npx jetify` call: https://github.com/BlueWallet/BlueWallet/blob/9417e068481765473de3086c47e8f19369b726ef/package.json#L43 may be removed (or not, but run without npx instead) as running `./node_modules/.bin/jetify` or `./node_modules/.bin/jetifier` return:
> Jetifier is deprecated. Future versions of react-native CLI may not run it by default.

also from https://www.npmjs.com/package/jetifier 
> Note that jetifier is included and ran automatically with react-native-community/cli for React Native versions 0.60 and above, so you do not need to install and run jetifier manually.

installation of related `jetifier` package in devDependencies:
https://github.com/BlueWallet/BlueWallet/blob/9417e068481765473de3086c47e8f19369b726ef/package.json#L27

2. The second npx usage is calling `npx detox` https://github.com/BlueWallet/BlueWallet/blob/9417e068481765473de3086c47e8f19369b726ef/package.json#L50 while the detox package installed as part of dependencies (which add `./node_modules/.bin/detox` binary):
https://github.com/BlueWallet/BlueWallet/blob/9417e068481765473de3086c47e8f19369b726ef/package.json#L103

When using yarn package from alpine distro instead of npm, the npx package is not installed (can be installed by `yarn add npx`) this issue is about removing the `npx` usage as it can be avoided which decrease the amount of needed packages/deps to compile the app, currently errors will occur if it's not installed when running `yarn postinstall` after running `yarn install`

### PR DESCRIPTION
**Pull request by Overtorment**

### CODE REVIEW

**Comment by GladosBlueWallet** [comment]
♫ This was a triumph. I'm making a note here: HUGE SUCCESS ♫

 [android in browser] https://appetize.io/app/1zezux3r7c1chmy58zpj7nu5xc?device=pixel4

[download apk](http://filestorage.bluewallet.io:1488/remove-npx-1620233700.apk)

File: package.json [hunk 1]
```diff
@@ -40,15 +39,15 @@
...
     "maccatalystpatches": "./scripts/maccatalystpatches/applypatchesformaccatalyst.sh",
-    "e2e:debug-build": "detox build -c android.emu.debug",
-    "e2e:debug-test": "detox test -c android.emu.debug",
+    "e2e:debug-build": "./node_modules/.bin/detox build -c android.emu.debug",
+    "e2e:debug-test": "./node_modules/.bin/detox test -c android.emu.debug",
     "e2e:debug": "(test -f android/app/build/outputs/apk/debug/app-debug.apk && test -f android/app/build/outputs/apk/androidTest/debug/app-debug-androidTest.apk) || npm run e2e:debug-build; npm run e2e:debug-test",
-    "e2e:release-build": "npx detox build -c android.emu.release",
-    "e2e:release-test": "detox test -c android.emu.release --record-videos all --take-screenshots all --headless --loglevel trace",
+    "e2e:release-build": "./node_modules/.bin/detox build -c android.emu.release",
+    "e2e:release-test": "./node_modules/.bin/detox test -c android.emu.release --record-videos all --take-screenshots all --headless --loglevel trace",
```
Comment (limpbrains) [inline package.json:50]: If you are running package bin from `package.json` `scripts` section, you don't need to use `npx` or provide path, like `./node_modules/.bin/`

`"e2e:release-build": "detox build -c android.emu.release",`
will work without any prefix

**Review by limpbrains — CHANGES_REQUESTED** [review]

**Comment by GladosBlueWallet** [comment]
♫ This was a triumph. I'm making a note here: HUGE SUCCESS ♫

 [android in browser] https://appetize.io/app/7w4e3uck5rew3n0qm19znzce8g?device=pixel4

[download apk](http://filestorage.bluewallet.io:1488/remove-npx-1620243121.apk)

**Review by limpbrains — APPROVED** [review]

**Comment by GladosBlueWallet** [comment]
Unbelievable. You, [subject name here], must be the pride of [subject hometown here]!

**Pull request merged**